bentoml/_internal/frameworks/transformers.py

```diff
@@ -84,13 +87,11 @@ def _check_flax_supported() -> None:  # pragma: no cover
 class TransformersOptions(ModelOptions):
     """Options for the Transformers model."""
 
-    task: str = attr.field(
-        validator=[
-            attr.validators.instance_of(str),
-            lambda instance, attribute, value: transformers.pipelines.check_task(value),  # type: ignore
-        ]
-    )
-
+    task: str = attr.field(validator=[attr.validators.instance_of(str)])
+    tf: t.Tuple(str) = (attr.field(validator=[attr.validators.instance_of(str)], default=None),)  # type: ignore
+    pt: t.Tuple(str) = (attr.field(validator=[attr.validators.instance_of(str)], default=None),)  # type: ignore
```

Comment:
I believe you shouldn't doing this. Instead you should use factory=tuple and attrs.validators.deep_iterable

Reply:
Good catch. I forgot to update the type after changing the type to `tuple`.